lib/components/viewers/trip-viewer.js

```diff
@@ -159,30 +196,33 @@ class TripViewer extends Component {
                 }
 
                 return (
-                  <Stop key={i}>
-                    {/* the departure time */}
-                    <div className="stop-time">
-                      <DepartureTime
-                        originDate={startOfDay}
-                        stopTime={tripData.stopTimes[i]}
-                      />
-                    </div>
-
-                    {/* the vertical strip map */}
-                    <div className="strip-map-container">
-                      {highlightClass && <div className={highlightClass} />}
-                      <div className={stripMapLineClass} />
-                      <div className="strip-map-icon">
-                        <StyledIconWrapper>
-                          <Circle />
-                        </StyledIconWrapper>
-                      </div>
-                    </div>
+                  <Stop aria-label={stopLabel} key={i}>
```

Comment:
`aria-label` does not seem to be picked up on a regular `<li>`, probably because it is not interactive. I think it is better to insert a `<span>` before or after the stop name that is not visible to sighted users (e.g. use zero-width/height and/or clip).

Reply:
Smart, thank you!